### DISCUSSION TITLE
Build error due failing to run custom command for `tree-sitter`

### DISCUSSION THREAD

**Discussion by tachy0nnn** [zed-industries/zed#22535] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

When I am trying to build Zed for Windows with VS 2022 and all dependencies, I getting this error:
```
error: failed to run custom build command for `tree-sitter v0.23.2`

Caused by:
  process didn't exit successfully: `D:\cargoTargetDir\release\build\tree-sitter-58a932852bc8de03\build-script-build` (exit code: 1)
  --- stderr


  error occurred in cc-rs: Command "D:\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.42.34433\\bin\\HostX64\\x64\\cl.exe" "-nologo" "-MD" "-O2" "-Z7" "-Brepro" "-I" "D:\\cargoTargetDir\\release\\build\\wasmtime-c-api-impl-8af79d05b0b9e418\\out/include" "-I" "C:\\Users\\Alexander\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\tree-sitter-0.23.2\\src" "-I" "C:\\Users\\Alexander\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\tree-sitter-0.23.2\\src\\wasm" "-I" "C:\\Users\\Alexander\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\tree-sitter-0.23.2\\include" "-DTREE_SITTER_FEATURE_WASM=" "-Dstatic_assert(...)=" "-FoD:\\cargoTargetDir\\release\\build\\tree-sitter-0e899fdf8cf3684f\\out\\9d101ca0d5afdee2-lib.o" "-c" "C:\\Users\\Alexander\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\tree-sitter-0.23.2\\src\\lib.c" with args cl.exe did not execute successfully (status code exit code: 2).


warning: build failed, waiting for other jobs to finish...
```

### Zed Version and System Specs

- Zed 0.166.1
- Windows 11 23H2 X64, Build Number 22631.4602
- 16GB Ram

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary>

```

```
</details>

**Comment by Ratragon** [comment]
I meet the same problem...qwq

**Comment by lawzava** [comment]
This is usually an SDK mismatch with your host version. 

Using VS Installer -> Modify make sure that Windows SDK version matches your OS Build Number.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
